stable/rethinkdb/Chart.yaml

```diff
@@ -0,0 +1,13 @@
+name: rethinkdb
+description: The open-source database for the realtime web
+version: 0.0.1
+keywords:
+- rethinkdb
+- database
+- nosql
+home: https://www.rethinkdb.com
```

Comment:
I think the best place to pick a icon from is inside the repo -> https://github.com/rethinkdb/docs/tree/master/_jekyll/_images/api_illustrations it all released under https://creativecommons.org/licenses/by-sa/4.0/